protocol/contracts/beanstalk/metadata/MetadataImage.sol

```diff
@@ -524,13 +526,93 @@ contract MetadataImage {
                 useAsset(getTokenName(token), 240, 4),
                 '<rect x="0" y="330" width="255" height="20" rx="5" fill="#242424"/>',
                 movingTokenAddress(token),
-                '<text x="230" y="14.5" font-size="12" fill="White" text-anchor="end" font-family="futura">Stem: ',
-                uint256(stem).toString(),
+                '<text x="235" y="14.5" font-size="12" fill="White" text-anchor="end" font-family="futura">Stem: ',
+                SciNotation(stem),
                 '</text>'
             )
         );
     }
 
+    function SciNotation(int96 stem) internal pure returns (string memory) {
```

Comment:
Why does this function start with an uppercase character?

Reply:
mistake, fixed: c58be9b7fa3eda06805602e40e1e93f0fb281137